tests/python/release_checklist/check_overrides_2d.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import annotations
+
+import os
+from argparse import Namespace
+from uuid import uuid4
+
+import rerun as rr
+import rerun.blueprint as rrb
+
+README = """
+# 2D views: Overrides, logged values, defaults, fallbacks
+
+This checks that component and visualizer overrides behave as expected for 2D views.
+
+---
+
+* Start with view **1)**.
+    * Double-click any of the arrows to select the arrow batch.
+    * Remove the extra `Points2D` visualizer.
+    * Remove all extra component overrides on the remaining `Arrows2D` visualizer.
+    * You should end up with the exact same view as the one on the right of view **1)**, which itself should look exactly like view **2)**.
+
+For reference, here is what view **1)** should look like:\
+![View 1](https://static.rerun.io/check_overrides_2d_view_1/5454441272b0abee8dbc8d7a342ecfc4036cbe9b/480w.png)
```

Comment:
this should go at the start of the list here: That's what it looks like _before_ above actions are done by the tester